jupyter_notebook/notebookapp.py

```diff
@@ -203,6 +241,7 @@ def init_settings(self, ipython_app, kernel_manager, contents_manager,
             config=ipython_app.config,
             jinja2_env=env,
             terminals_available=False,  # Set later if terminals are available
+            log=log,
```

Comment:
I was mainly asking because we log everywhere, and don't need access in
settings, so I was dubious that it actually needed to be added.

On Mon, Apr 20, 2015 at 5:16 PM, Jonathan Frederic <notifications@github.com
> wrote:

> In jupyter_notebook/notebookapp.py
> <https://github.com/jupyter/jupyter_notebook/pull/34#discussion_r28741120>
> :
>
> > @@ -203,6 +241,7 @@ def init_settings(self, ipython_app, kernel_manager, contents_manager,
> >              config=ipython_app.config,
> >              jinja2_env=env,
> >              terminals_available=False,  # Set later if terminals are available
> > +            log=log,
>
> [image: :heavy_check_mark:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter_notebook/pull/34/files#r28741120>.
>